src/include/reconcile.h

```diff
@@ -138,6 +138,12 @@ struct __wt_reconcile {
      */
     bool all_empty_value, any_empty_value;
 
+    /*
+     * After we've built the page image in reconciliation and we are intending to insert updates
+     * into the history store, any error will result in a panic.
+     */
+    bool panic_on_error;
```

Comment:
I asked for this change here, but on seeing the code change I think it would be better to split it into a different ticket. That way we can choose to selectively revert and/or backport the change.

Reply:
Will undo this change and have created WT-8074 to track the panic work.